setup.py

```diff
@@ -20,10 +23,24 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
```

Comment:
Well, I agree with your suggestion, so I'll go through with it :D I agree that comments are important, and especially the absence of misleading comments.